platform/ethereum/ens.go

```diff
@@ -0,0 +1,46 @@
+package ethereum
```

Comment:
rename `ens.go` to `ns.go`

Reply:
ens is fine, we probably will have `.crypto` resolving, which is not ens powered